### PR TITLE
Configure timezone for Grafana dashboards

### PR DESCRIPTION
**Pull request by nlamirault**
Signed-off-by: Nicolas Lamirault <nicolas.lamirault@gmail.com>

### CODE REVIEW

File: dashboards/defaults.libsonnet [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+
   local grafanaDashboards = super.grafanaDashboards,
 
   // Automatically add a uid to each dashboard based on the base64 encoding
   // of the file name and set the timezone to be 'default'.
   grafanaDashboards:: {
     [filename]: grafanaDashboards[filename] {
       uid: std.md5(filename),
-      timezone: 'UTC',
+      timezone: '%(grafanaTimezone)s' % $._config.grafanaK8s,
```
Comment (brancz) [inline dashboards/defaults.libsonnet:10]: why not just:

```
timezone: $._config.grafanaK8s.grafanaTimezone
```

?

Comment (nlamirault) [reply dashboards/defaults.libsonnet]: @brancz due to a newbie in Jsonnet :)

**Comment by brancz** [comment]
hmm I'm actually not sure we can rely on the global here ... @paulfantom any suggestions on how to solve this?

**Comment by brancz** [comment]
For a start just use a local reference like I did here: https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/627

And then we can refactor the configuration to be closer to their usage/"call-sites" in follow ups.

**Review by paulfantom — APPROVED** [review]
lgtm

**Pull request merged**